scipy/stats/_continuous_distns.py

```diff
@@ -661,6 +662,66 @@ def _munp(self, n, c, d):
 burr = burr_gen(a=0.0, name='burr')
 
 
+class burr12_gen(rv_continuous):
+    """A Burr (Type XII) continuous random variable.
+
+    %(before_notes)s
+
+    See Also
+    --------
+    fisk : a special case of either `burr` or ``burr12`` with ``d = 1``
+    burr : Burr Type III distribution
+
+    Notes
+    -----
+    The probability density function for `burr` is::
+
+        burr12.pdf(x, c, d) = c * d * x**(c-1) * (1+x**(c))**(-d-1)
+
+    for ``x > 0``.
+
+    `burr12` takes ``c`` and ``d`` as shape parameters.
+
+    This is the PDF corresponding to the twelfth CDF given in Burr's list;
+    specifically, it is equation (20) in Burr's paper [1]_.
+
+    %(after_notes)s 
+    
+    The Burr type 12 distribution is also sometimes referred to as
+    the Singh-Maddala distribution from NIST.
```

Comment:
@josef-pkt has mentioned this in this [comment](https://github.com/scipy/scipy/pull/5654#issuecomment-168397329). From a simple search [this](http://www.itl.nist.gov/div898/software/dataplot/refman2/auxillar/b12pdf.htm) link came up. Would this be good ?